tests/api2/test_190_filesystem.py

```diff
@@ -277,3 +278,89 @@ def test_09_test_dosmodes():
                 results = POST('/filesystem/get_dosmode', p)
                 assert results.status_code == 200, results
                 assert results.json() == expected_flags
+
+
+def test_10_acl_path_execute_validation():
+    ds_name = 'acl_execute_test'
+    target = f'{pool_name}/{ds_name}'
+    path = f'/mnt/{target}'
+
+    NFSV4_DACL = [
+        {'tag': 'owner@', 'id': -1, 'type': 'ALLOW', 'perms': {'BASIC': 'FULL_CONTROL'}, 'flags': {'BASIC': 'INHERIT'}},
+        {'tag': 'group@', 'id': -1, 'type': 'ALLOW', 'perms': {'BASIC': 'FULL_CONTROL'}, 'flags': {'BASIC': 'INHERIT'}},
+        {'tag': 'USER', 'id': 65534, 'type': 'ALLOW', 'perms': {'BASIC': 'FULL_CONTROL'}, 'flags': {'BASIC': 'INHERIT'}},
+        {'tag': 'GROUP', 'id': 65534, 'type': 'ALLOW', 'perms': {'BASIC': 'FULL_CONTROL'}, 'flags': {'BASIC': 'INHERIT'}},
+    ]
+
+    # Do NFSv4 checks
+    with create_dataset(target, {'acltype': 'NFSV4', 'aclmode': 'PASSTHROUGH'}, None, 770):
+        sub_ds_name = f'{ds_name}/sub'
+        sub_target = f'{pool_name}/{sub_ds_name}'
+        sub_path = f'/mnt/{sub_target}'
+
+        """
+        For NFSv4 ACLs four different tags generate user tokens differently:
+        1) owner@ tag will test `uid` from payload
+        2) group@ tag will test `gid` from payload
+        3) GROUP will test the `id` in payload with id_type
+        4) USER will test the `id` in mayload with USER id_type
+        """
+
+        # Start with testing denials
+        with create_dataset(sub_target, {'acltype': 'NFSV4', 'aclmode': 'PASSTHROUGH'}):
+            acl = deepcopy(NFSV4_DACL)
+            names = ['daemon', 'apps', 'nobody', 'nogroup']
+            for idx, entry in enumerate(NFSV4_DACL):
```

Comment:
From here I see indentation issues there is one extra space on almost all lines below.